src/components/modules/blog/cta/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { StyledBlogCTASubtitle, StyledBlogCTAWrapper } from './cta.styles';
+import type { BlogCTAProps } from './cta.types';
+
+import { Button } from '@/components/elements/button';
+import { Heading } from '@/components/elements/heading';
+import { CaretRight } from '@/components/elements/icon';
+
+export const BlogCTA = ({ title, buttonText, subtitle, buttonHref }: BlogCTAProps) => {
+  return (
+    <StyledBlogCTAWrapper>
+      <StyledBlogCTASubtitle>{subtitle}</StyledBlogCTASubtitle>
+      <Heading fontSize="m" color="textGradient">
```

Comment:
@RBoraurito, the default heading tag is H1. Shouldn't this be an h3 or something?

Reply:
Think the same, it should be an H3